src/gui/mainwindow.ui

```diff
@@ -248,33 +248,45 @@
     <string>Set Global Upload Limit...</string>
    </property>
   </action>
-  <action name="actionBottomPriority">
+  <action name="actionBottomQueuePos">
    <property name="text">
-    <string>Minimum Priority</string>
+    <string>Bottom Queue</string>
```

Comment:
IMO, "Bottom Position" and so on...

Reply:
@thalieht, @glassez:  "Position" seems pretty generic, but… 

This one is somewhat tricky, because there is no adjective form (or standard adjective terminology) for the English word `queue`.  

The shortest formal phrasing I can think of (involving "queue") is: 

>Top of Queue 
>Raise in Queue 
>Lower in Queue 
>Bottom of Queue

Reply:
This is not much better IMO because one may wonder Bottom Position of what? Unless we rely on the tooltip. Grammatically it's definitely better though. I'll wait a while in case there is some better proposition.

Reply:
![capture4](https://user-images.githubusercontent.com/6451685/49791140-794ebf80-fd38-11e8-95a2-460516c68076.JPG)
I suppose i could get rid of the different tooltips with this because these are pretty self explanatory.